code/src/index.css

```diff
@@ -1,13 +1,111 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  margin: 10px;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(114, 209, 181);
```

Comment:
Good work ashwin!!! I understand the code and like that you go for higher level than blue!!!

- [ YES]  Am I able to understand the code easily?
- [ NO]  Does the code have lots of duplicates? Could it be tidied up?
- [ NO ]  Does it work or does anything look broken?
- [ YES]  Does it follow the general and blue requirements?